### ISSUE TITLE
It should be possible to run Karma  test using Chrome

### ISSUE THREAD

**Issue by jensborch** [Nykredit/gript#67]
Primarily for debugging purpose it should be possible to run Karma  test in Chrome. This could be enabled using a configuration setting gulpfile.js or it could be a separate task.

**Comment by jotka** [comment]
I'm going to move the default Karma.js config  into the `test` task itself and then give you the ability to override Karma settings in the `gulpfile.js`.  At the same time, I will get rid of `karma.conf.js`.

**Comment by jotka** [comment]
from the upcoming readme.md:


## Testing
Gript will automatically run your tests using [Karma](https://karma-runner.github.io). Your tests can be written in JavaScript or TypeScript (they will be compiled first). Tests filenames must end in `test` or `Test` (for example `PortfolioServiceTest.ts`, `PortfolioService_test.ts`, `portfolioService_test.js`).
Your tests will be executed using headless, [PhantomJS](http://phantomjs.org) browser. You can customize this behavior altogether with other Karma options in the `karma` section of the `gulpfile.js`.
For example, if you need to run your tests under Chrome browser, amend the default configuration:
 
 ```
 karma: {
         frameworks: ['jasmine', 'sinon', 'angular-filesort'],
         plugins: [
             'karma-jasmine',
             'karma-sinon',
             'karma-angular-filesort',
             'karma-phantomjs-launcher',
             'karma-chrome-launcher',
             'karma-junit-reporter',
             'karma-coverage',
             'karma-ng-html2js-preprocessor'
         ],
         browsers: ['Chrome']
     }



release today.

**Comment by jotka** [comment]
released as 0.0.48
